### ISSUE TITLE
custom sticks in evdev broken by #3817

### ISSUE THREAD

**Issue by toccata10** [RPCS3/rpcs3#3962]
With #3817, the directions for my custom stick (based on a DragonRise board) can't be used anymore.
It could be because of different ranges: with an xbox 360 controller: it goes from -32767 to +32768, whereas with the dragonrise, it goes from 0 to 255.
A screenshot of the configuration window with my non-moving stick showing a big offset (only left analog is important):
![selection_053](https://user-images.githubusercontent.com/7478580/34261831-c4eaa4c6-e66a-11e7-8fea-e9ab614686bd.jpg)

**Comment by Megamouse** [comment]
buy another gamepad

**Comment by Megamouse** [comment]
I get the feeling this would be much easier if they would add negative values to the dragonrise drivers

**Comment by toccata10** [comment]
Yes, sure.
A workaround for me is to comment these lines (in 2 places) to use the Dragonrise
```
		// Triggers do not need handling of negative values
//		if (min >= 0)
//		{
//			float fvalue = ScaleStickInput(val, min, max);
//			button_values.emplace(code, std::make_pair<u16, bool>(static_cast<u16>(fvalue), false));
//			continue;
//		}
```
3 ideas: as @Megamouse suggested, add a button 

> a button like "handle trigger axis" 
or it could be in a configuration file (with the gamepad name) like:
```
DragonRise: true
Microsoft Xbox360: false
```
or a parameter in the config file:
handle trigger axis: true

**Comment by Megamouse** [comment]
can be closed?

**Comment by toccata10** [comment]
Closed thanks to #3974
